roles/openshift_node/templates/openshift.docker.node.service

```diff
@@ -34,6 +34,7 @@ ExecStart=/usr/bin/docker run --name {{ openshift.common.service_type }}-node \
   -v /lib/modules:/lib/modules -v /etc/origin/openvswitch:/etc/openvswitch \
   -v /etc/origin/sdn:/etc/openshift-sdn -v /var/lib/cni:/var/lib/cni \
   -v /etc/systemd/system:/host-etc/systemd/system -v /var/log:/var/log \
+  {% if openshift_use_nuage | default(false) -%} $NUAGE_ADDTL_BIND_MOUNTS {% endif -%} \
```

Comment:
I think '-%}' above may be in error.  No need for the '-' character in the if/endif blocks.

Reply:
My 50c guess, the `-` will remove the extra NL which should have no effect on the one-line expression.